api/conf/urls/openapi.py

```diff
@@ -1,4 +1,5 @@
 from django.urls import path
+from django.views.generic.base import TemplateView
```

Comment:
This import should be removed so that lint check can pass. `TemplateView` is not used in the file so the linter complains that it should not be imported.